### ISSUE TITLE
How add project to project.json?

### ISSUE THREAD

**Issue by xerron** [dunstontc/projectile.nvim#3]
Need more documentation.

I am try:

`:Denite projectile:add `

**Comment by amireldor** [comment]
Yes, please show more documentation. I tried digging into Denite's own documentation but that's obscure as well.

**Comment by dunstontc** [comment]
Currently you've got to hit tab in Denite's normal mode

![add-project](https://raw.githubusercontent.com/dunstontc/assets/master/images/projectile-add-project.png)

But I've been using this mapping:
```vim
 call denite#custom#map(
       \ 'normal',
       \ 'a',
       \ '<denite:do_action:add>',
       \ 'noremap'
       \)
```
then using `a` to prompt for the addition.

But I'll see if I can figure out how to enable adding it as an argument.
